CHANGELOG.md

```diff
@@ -1,5 +1,29 @@
 # @digitalbazaar/cborld ChangeLog
 
+## 8.0.0 - 2024-mm-dd
+
+### Added
+- Add TypedLiteral, UntypedLiteral encoders and decoders to more generally
+  handle type-specific behavior such as cryptosuite string encoding as well
+  as pass through uncompressible untyped values.
+
+### CHANGED
+- Restructure term registry system to be more general with four tables which
+  can all be passed by the user during encoding and decoding:
+    - keywordsTable: for JSON-LD keywords (e.g. '@context', '@type')
+    - stringTable: for JSON-LD string values. This subsumes 'appContextMap',
```

Comment:
Maybe `appContextMap` should only be allowed in `legacy` mode? And users of the library don't have to know so much about this then. They are either running in legacy mode and use `appContextMap` or they are running in the new mode and they use the new tables instead.